articles/time_to_say_goodby.md

```diff
@@ -0,0 +1,55 @@
+ 
+3. Redefine roles, responsibilities and meetings in the team and communicate the result
+With the new phase and activities, responsibilities will change, too. Maybe a business analyst will no longer be responsible for requirements only but also for training and support of the support team or a test manager will no longer be needed. Some tasks shall be done by operations now and no longer by the project team itself. Therefore, a new definition of roles and responsibilities in the team is crucial as well as communication of the new assignments. Team meetings or defect meetings may no longer be required. Therefore, meetings to discuss existing production issues, progress within trainings or performance and usage statistics are more relevant. From my perspective it is better to stop old meeting series and setup new meetings with new agenda and participants instead of reusing old ones to have an official shift.
+ 
+4. Look for contact with the team member and their Superior
+Sometimes team members will be lured to directly change to another project. Sometimes this is not the case. Anyway, as a project manager I recommend to discuss with the superiors by when the team member is leaving the current project and to which share the person will work in the new project or can take over new tasks outside the current project. This agreement should be taken between Superior, project manaaer(ers) and team member in order that all have the same view of the situation.
+ 
+5. No project without any learnings
+Invite right beforehand all team members to a lessons learned meeting. Ensure that the meeting will either be split in separate ones if there are too many people involved for a common retro or search for someone to support you doing the lessons learned workshop. The workshop may be organized as a “world café”. Good practice, from my experience, is to make the project stages visible by putting cards on the wall for each month and each event during the project duration. I also saw such a review in way of a collage or story / poem. As a source of information, you can use the monthly status reports or your project diary. You will be amazed of what happened (and you already forgot about it) and what you and the team achieved. One important point for me is, that at the end of the meeting every single participant has the possibility to write his own lessons learned and take these notes with him into the next project. Maybe you can prepare some cards beforehand and hand them out as template that they can fill out. You can also ask your team members either for feedback to your job or if they want you to provide them feedback about themselves. To be prepared for these activities it is helpful to start writing a project diary and note some keywords each week what happened, what was good, what was bad, what are my learnings and also with your private project diary what happened to all team members so that you can see their development through the time being.
```

Comment:
Knackiger: "Good practice, from my experience, is to make the pr" -> "Good practice is to make the pr"

"I also saw such a review in way of a collage or story / poem. " -> Da sehe ich jetzt den Infogehalt nicht so klar. Weglassen?

"Maybe you can prepare some cards beforehand and hand them out as template that they can fill out. " -> Wenig Infogehalt, weglassen?